### PR TITLE
Arbts multicall

### PR DESCRIPTION
**Pull request by fredlacs**

### CODE REVIEW

**Review by DZGoldman — APPROVED** [review]
love it, LGTM pending conflicts

**Comment by fredlacs** [comment]
This contains a breaking change to the `init` function so we should prob do a major version bump?

File: packages/arb-ts/src/lib/bridge.ts [hunk 1]
```diff
@@ -850,4 +858,40 @@ export class Bridge {
       this.l2Provider
     )
   }
+
+  public async getTokenBalanceBatch(
+    userAddr: string,
+    tokenAddrs: Array<string>,
+    targetNetwork: 'L1' | 'L2'
+  ) {
```
Comment (hkalodner) [inline packages/arb-ts/src/lib/bridge.ts:866]: Can we add typing to the return value here

**Review by hkalodner — CHANGES_REQUESTED** [review]

File: packages/arb-ts/src/lib/bridge.ts (hunk 1 above)
Comment (fredlacs) [reply packages/arb-ts/src/lib/bridge.ts:866]: This was added in the follow up PR
https://github.com/OffchainLabs/arbitrum/pull/1673

Comment (fredlacs) [reply packages/arb-ts/src/lib/bridge.ts:866]: https://github.com/OffchainLabs/arbitrum/pull/1673/commits/62248797b99b70552b0a24f93bc9bb4372140a48

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**